### DISCUSSION TITLE
dist folder

### DISCUSSION THREAD

**Discussion by DavidGalicia** [cdmoro/bootstrap-vue-3#415] [Q&A]
After I use git to clone bootstrap-vue-3, do I need to run "npm run build" to generate a "dist" folder? I noticed the cloned project doesn't come with this folder but when the project is obtained using "npm -i" then the project comes with the folder.

**Comment by mariusa** [comment]
Yes, that should create the `dist` folder.

**Reply by DavidGalicia** [reply]
Hi and thank you! The `npm run build` command generated errors and didn't create a `dist` folder. I made a new issue #418 to find out why this happened.
